ledger-service/http-json/src/it/scala/http/WebsocketServiceIntegrationTest.scala

```diff
@@ -3,942 +3,16 @@
 
 package com.daml.http
 
-import akka.NotUsed
```

Comment:
It might also disable the test. There were no changes to the moved class. And maybe one day we of the software dev community will, finally, collectively regret the decision to anoint git the great successor VCS.